python3-aliyun-img-utils.spec

```diff
@@ -15,31 +15,38 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
-
-Name:           python3-aliyun-img-utils
+%{?!python_module:%define python_module() python-%{**} python3-%{**}}
+%if 0%{?suse_version} >= 1500
+%define skip_python2 1
```

Comment:
Python2 builds should be skipped. I don't think we build the dependencies as python2 either.

Reply:
This is not a new change. The goal with this PR was to sync the spec file from the build service.

Reply:
OK, let's get the conflicts resolved and get it merged then.